src/index.js

```diff
@@ -30,4 +31,5 @@ export {
   isNullOrWhitespace,
   startsWith,
   removeDuplicates,
+  additon,
```

Comment:
I would prefer that this were called `add` instead of `addition`. Care to make the update?